### PR TITLE
implemented TodoList, TodoInfo, UserInfo components

### PR DESCRIPTION
**Pull request by SerhiiRomaniukWD**
[DEMO LINK](https://serhiiromaniukwd.github.io/react_static-list-of-todos/)

### CODE REVIEW

File: src/components/TodoList/TodoList.tsx [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <ul className="TodoList">
+    {todos.map((todoPer) => (
```
Comment (Yevheniya-Deryka) [inline src/components/TodoList/TodoList.tsx:11]: what is "todoPer"? )

**Review by Yevheniya-Deryka — APPROVED** [review]
GJ!

File: src/components/TodoList/TodoList.tsx (hunk 1 above)
Comment (SerhiiRomaniukWD) [reply src/components/TodoList/TodoList.tsx:11]: i will change it)

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    completed,
+    title,
+    user,
+    id,
+  } = todo;
+
+  return (
+    <article key={id} className={`TodoInfo ${completed && 'TodoInfo--completed'}`}>
```
Comment (Denchepornyuk) [inline src/components/TodoInfo/TodoInfo.tsx:19]: do you need `key` here?

Comment (Denchepornyuk) [inline src/components/TodoInfo/TodoInfo.tsx:19]: if you want to add class with some condition it's better to use classnames package
https://www.npmjs.com/package/classnames

File: src/components/TodoList/TodoList.tsx [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <ul className="TodoList">
+    {todos.map((todoBlock) => (
+      <li key={todoBlock.id}>
+        <TodoInfo todo={todoBlock} />
+      </li>
```
Comment (Denchepornyuk) [inline src/components/TodoList/TodoList.tsx:14]:
```suggestion
    {todos.map(todo => (
      <li key={todo.id}>
        <TodoInfo todo={todo} />
      </li>
```

**Review by denys-cheporniuk — CHANGES_REQUESTED** [review]

File: src/components/TodoInfo/TodoInfo.tsx [hunk 4]
```diff
@@ -1,4 +1,27 @@
...
+  const {
+    completed,
+    title,
+    user,
+  } = todo;
+
+  return (
+    <article className={
+      classNames('TodoInfo', completed && 'TodoInfo--completed')
+    }
```
Comment (OleksiiBlezniuk) [inline src/components/TodoInfo/TodoInfo.tsx:21]:
```suggestion
    <article className={classNames('TodoInfo', {
        'TodoInfo--completed': completed,
    })}
```

**Review by OleksiiBlezniuk — APPROVED** [review]

File: src/components/TodoInfo/TodoInfo.tsx (hunk 4 above)
Comment (SerhiiRomaniukWD) [reply src/components/TodoInfo/TodoInfo.tsx:21]: okey, i will fix it

**Pull request closed without merging**